webapp/templates/rewriterule-create.xhtml

```diff
@@ -0,0 +1,176 @@
+<html version="XHTML+RDFa 1.0" xmlns="http://www.w3.org/1999/xhtml"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:calli="http://callimachusproject.org/rdf/2009/framework#">
+<head>
+    <title>New Rewrite Rule</title>
+    <link rel="help" href="../../callimachus-reference#PURL" target="_blank" title="Help" />
+    <style>
+        form#form textarea.pattern {
+            white-space: pre;
+            word-wrap: normal;
+        }
+        #ruleTable th, #ruleTable td { 
+            padding-left: 4px; 
+            padding-right: 4px; 
+        }
+        #ruleTable>thead>tr:first-child>th { 
+            text-align: center;
+        }
+        #ruleTable>thead { 
+            border-bottom:1px solid black;
+        }
+        #ruleTable>thead>tr:nth-child(1)>th:nth-child(1), 
+        #ruleTable>thead>tr:nth-child(2)>th:nth-child(2),
+        #ruleTable td:nth-child(2) { 
+            border-right:1px solid black;
+        }
+        .selectize-input {
+            margin-top:4px;
+        }
+    </style>
+    <script type="text/javascript" src="../scripts/purl.js"></script>
+</head>
+<body>
+    <form role="form" id="form" method="POST" action="" enctype="text/turtle" typeof="calli:RewriteRule" class="container">
+        <div class="page-header">
+            <h1>New Rewrite Rule</h1>
+        </div>
+        <div class="row">
+            <div class="col-sm-6">
+                <fieldset>
+                    <div class="form-group">
+                        <label for="local">Local name</label>
+                        <div>
+                            <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
+                                onchange="calli.updateProperty(event, 'rdfs:label')" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="comment">Comment</label>
+                        <div>
+                            <textarea id="comment" class="form-control"
+                                onchange="calli.updateProperty(event, 'rdfs:comment')">{rdfs:comment}</textarea>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <div class="col-sm-6">
+                <aside class="well">
+                    <p>PURLs are used to curate the URL resolution process, thus solving
```

Comment:
This help text should include a description of what the template field is used for and what format the template field should take.